tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Builders/OrganisationControllerBuilder.cs

```diff
@@ -53,18 +53,28 @@ internal OrganisationControllerBuilder WithUpdateOrganisation(Organisation organ
             return this;
         }
 
-        public OrganisationControllerBuilder WithOrganisationRepository(IOrganisationRepository organisationRepository)
+        internal OrganisationControllerBuilder WithOrganisationRepository(IOrganisationRepository organisationRepository)
         {
             _organisationRepository = organisationRepository;
             return this;
         }
 
-        public OrganisationControllerBuilder WithCreateOrganisationServiceReturningResult(bool isSuccess, string resultValue)
+        internal OrganisationControllerBuilder WithCreateOrganisationServiceReturningSuccess(Guid result)
+        {
+            return WithCreateOrganisationServiceReturningResult(Result.Success(result));
+        }
+
+        internal OrganisationControllerBuilder WithCreateOrganisationServiceReturningFailure(string result)
+        {
+            return WithCreateOrganisationServiceReturningResult(Result.Failure<Guid>(new ErrorDetails(result)));
+        }
+
+        internal OrganisationControllerBuilder WithCreateOrganisationServiceReturningResult(Result<Guid> result)
```

Comment:
I'd probably make this `private` to encourage the use of `WithCreateOrganisationServiceReturningSuccess` and `WithCreateOrganisationServiceReturningFailure`. Do you think it might be useful exposing it?

Reply:
Typo (of sorts), it should be private here. Leaving it internal will be confusing.